[8.2][DOCS] Adds ML-related items to What's new section (#1889)
elastic/security-docs: docs/whats-new.asciidoc

In addition to improvements to indicator match rule performance, the default indicator index query for custom and prebuilt {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match rules] has been updated to `@timestamp > "now-30d/d"` to fine-tune the query range and performance.

> *Deploy DGA and Living-off-the-land supervised models in Fleet*

Reviewer: ```suggestion
*Deploy DGA and Living-off-the-land supervised models in {fleet}*
```

> Incorporating supervised models into integration packages provides you a more

Reviewer: ```suggestion
Incorporating supervised models into integration packages allows you to
```

> seamless experience to install package artifacts inside {kib} with a single

Reviewer: ```suggestion
seamlessly install package artifacts inside {kib} with a single
```

click. Now you can deploy

> https://www.elastic.co/blog/supervised-and-unsupervised-machine-learning-for-dga-detection[DGA]

Reviewer 2: Since that's the old 2020 blog, I think it's better to link to the new integration docs:
```suggestion
https://docs.elastic.co/en/integrations/dga[DGA]
```

> and https://www.elastic.co/blog/problemchild-detecting-living-off-the-land-attacks[Living-off-the-land (LotL)]

Reviewer 2: Ditto re newer content link:
```suggestion
and https://docs.elastic.co/en/integrations/problemchild[Living-off-the-land (LotL)]
```
